packages/studio-be/src/studio/cloud/cloud-client.ts

```diff
@@ -0,0 +1,125 @@
+import axios from 'axios'
+import { Logger } from 'botpress/sdk'
+import { isCDMError } from 'common/errors'
+import FormData from 'form-data'
+import { constants } from 'http2'
+
+import _ from 'lodash'
+import qs from 'qs'
+import { Result, Ok, Err } from 'ts-results'
+import { VError } from 'verror'
+import { CDMConflictError, UnexpectedError } from './errors'
+import { Bot, Introspect, OAuthAccessToken, PersonalAccessToken, Principals } from './types'
+
+export const MAX_BODY_CLOUD_BOT_SIZE = 100 * 1024 * 1024 // 100 MB
+
+export class CloudClient {
+  constructor(private logger: Logger) { }
+
+  public async createBot(props: {
+    personalAccessToken: string
+    name: string
+    workspaceId: string
+  }): Promise<Result<Bot, CDMConflictError | UnexpectedError>> {
+    const { personalAccessToken, name, workspaceId } = props
+    try {
+      const { data } = await axios.post(
+        `${process.CLOUD_CONTROLLER_ENDPOINT}/v1/bots`,
```

Comment:
Fixed in 9cf04b5

Unfortunately the PAT changes on each request and therefore cannot be passed as a constructor param, since the `CloudClient` is only constructed once at the server startup